.eslintrc.yml

```diff
@@ -32,182 +32,328 @@ parserOptions:
   unicodeCodePointEscapes: true
 
 rules:
+  # https://github.com/prettier/eslint-plugin-prettier#installation
   prettier/prettier: error
 
+  # `eslint-plugin-flowtype` rule list based on `v2.50.0`
+  # https://github.com/gajus/eslint-plugin-flowtype#eslint-plugin-flowtype
+
+  #flowtype/array-style-complex-type: undecided
```

Comment:
@mjmahone now all new rules are marked as `undecided` to prevent confusion

Reply:
Nice! Thank you